test/unit/api/api-get-linking-metadata.test.js

```diff
@@ -16,7 +16,12 @@ tap.test('Agent API - getLinkingMetadata', (t) => {
   let api = null
 
   t.beforeEach((done) => {
-    agent = helper.loadMockedAgent()
+    // implicitly disabling distributed tracing to match original config base settings
+    agent = helper.instrumentMockedAgent({
+      distributed_tracing: {
+        enabled: false
+      }
+    })
```

Comment:
Minor: but in a few months, etc. we won't understand what "original config base settings" were and why they would matter here.

I might have left as-is and updated the individual test cases to disable DT when they wanted DT disabled. Whether per test or special setup.